src/otx/v2/adapters/torch/mmengine/mmdet/modules/models/heads/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""Initial file for mmdetection heads."""
+# Copyright (C) 2023 Intel Corporation
```

Comment:
Can you check there is a new line between the module docstring and the license header for the rest of the files?

Reply:
I'll do that, but honestly, this kind of works should be done by pre-commit